changes/README.md

```diff
@@ -0,0 +1,62 @@
+# Fragments for the changelog
+
+This directory contains fragments for the future [NEWS](../NEWS.md) file.
+
+## Introduction
+
+We use <code>[towncrier]</code> to produce useful, summarized news files.
+
+There are 3 sections types:
+
+- API: `changes/api`
+- Tools: `changes/tools`
+- Build System: `changes/build`
+
+There are 3 news fragments types:
+
+- Breaking changes: `.breaking`
+- New: `.feature`
+- Fixes: `.bugfix`
+
+[towncrier]: https://pypi.org/project/towncrier/
+
+## Adding a fragment
+
+Add a short description of the change in a file `changes/SECTION/ID.FRAGMENT.md`,
+where:
+
+- `SECTION` and `FRAGMENT` values are described in the [previous section](#introduction).
+- `ID` is the corresponding issue identifier on Github, if relevant. If there is
+  no such issue, then `ID` should start with `+` and some identifier that make
+  the file unique in the directory.
+
+Examples:
+- A _bug fix_ for the _issue #463_ is an _API_ change, so the corresponding file
```

Comment:
I would recommend using something like `changes/api/463-fix-blah-add-xkb_foo.bugfix.md` so it's easier to spot without knowing all the issue numbers by heart (or looking into each file). But I think that could also be something we figure out over time.

Reply:
Agreed. I’ll check this tomorrow.

Reply:
So, the part preceding `.bugfix.md` can be anything, but currently I use it to add a link to the corresponding ticket. Unfortunately, `towncrier` does not handle the file name convention you propose.

So we can either:
- Only use ticket ID, as proposed: `123.bugfix.md`. We get the link to the issue. 
- Prefix with `+`, then optionally set ticket, then some short description:  `+123-description.bugfix.md`. No link to the issue.

We did not have links to issues until now. On one hand the second option is conservative and is more readable. On the other hand, these fragments should be written in the MR corresponding to the change, so it is unlikely we need to review them afterwards.

Reply:
Honestly, this isn't a huge deal so whatever makes life simpler for us (which seems to be the `123.bugfix.md` approach). Thanks for checking though!